static/app/views/onboarding/components/genericFooter.tsx

```diff
@@ -13,7 +13,7 @@ const GenericFooter = styled(motion.div)`
   display: flex;
   background-color: ${p => p.theme.background};
   justify-content: space-between;
-  box-shadow: 0px -4px 24px rgba(43, 34, 51, 0.08);
```

Comment:
Apologies for the stray cleanup, but moved this to a themed `dropShadowHeavyTop` variable since ACI will be using this also